### ISSUE TITLE
[Bug]: Aperture cannot be initialized with vertices

### ISSUE THREAD

**Issue by Kilichbek** [psychopy/psychopy#5926]
### PsychoPy Version

2023.2.2

### What OS are your PsychoPy running on?

macOS Silicon

### Bug Description

Encountering error during aperture initialization:
```
aperture = visual.Aperture(
            self.window, 
            shape=[(-0.02, -0.0), (-.8, .2), (0, .6), (.1, 0.06), (.8, .3), (.6, -.4)]
        )
```

> "/Applications/PsychoPy.app/Contents/Resources/lib/python3.8/psychopy/visual/aperture.py", line 96, in __init__
>     if shape in knownShapes:
> TypeError: unhashable type: 'list'

### Expected Behaviour

expected a shape with corresponding vertices for aperture

### Steps to Reproduce

1. just try to initialize aperture with vertices list

**Comment by miguelteixeira123456789** [comment]
I would like to work on this issue. Can you please assign it to me?

**Comment by miguelteixeira123456789** [comment]
I'm afraid that this issue is proving a bit too much for me, so I have to throw in the towel. I don't want to hold this up any further so I am unassigning it from myself. Please accept my apologies for any inconvenience I may have caused.

**Comment by TEParsons** [comment]
Fixed in #6303
